### PR TITLE
Databases week1

### PR DESCRIPTION
**Pull request by mehmetnuricavdar**

### CODE REVIEW

File: databases/week1/week1_HW.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+SELECT count(id)
+FROM task
+where due_date is null;
+
+-- 3 Find all the tasks that are marked as done
+
+select title, status_id
+from task
+where status_id='3';
+
```
Comment (Elia-Git) [inline databases/week1/week1_HW.sql]: How do I / some one else know status_id= 3 means Done, 

```
SELECT title,status.name 
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name="Done";
```

File: databases/week1/week1_HW.sql [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+-- 4 Find all the tasks that are not marked as done
+
+select title, status_id
+from task
+where status_id = '1' or status_id = '2';
+
```
Comment (Elia-Git) [inline databases/week1/week1_HW.sql]: Probably you only know what status_id, 1& 2 means

Try this 
```
SELECT title,status.name 
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name !="Done";
```

File: databases/week1/week1_HW.sql [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+-- 7 Get the title and due date of all tasks where the title or description contains database
+
+select title
+, due_date
+from task
+ where
+(title like '%database%')
+ or
+(title like '%database%');
```
Comment (Elia-Git) [inline databases/week1/week1_HW.sql]: line # 47 can be modified as 
`(description like %database%);`

File: databases/week1/week1_HW.sql [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+select status.name, count(task.status_id) as status_count
+from task
+    join status on task.status_id = status.id
+group by status.name;
+
+-- 10 Get the names of all statuses, sorted by the status with most tasks first
+select status.name, count(task.status_id) as status_count
+from task
+    join status on task.status_id = status.id
```
Comment (Elia-Git) [inline databases/week1/week1_HW.sql]: All good here!

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Please consider the given feedback and remove all empty line codes.
(like lines #2, 4,7 ...15,19 etc)

File: databases/week1/week1_HW.sql (hunk 1 above)
Comment (mehmetnuricavdar) [reply databases/week1/week1_HW.sql]: Hi Elia, Thank you so much for 

> 

Hi Elia. Thank you so much for reviewing my HW. I did the empty lines for readability but i am removing them.

File: databases/week1/week1_HW.sql (hunk 3 above)
Comment (mehmetnuricavdar) [reply databases/week1/week1_HW.sql]: ahhh yes i missed that part 🤦🏽‍♂️

**Pull request merged**